src/pages/ErrorPage/ErrorPage.component.tsx

```
@@ -0,0 +1,22 @@
+import { useLocation } from 'react-router-dom';
+
+import { Box } from '@mui/material';
+
+import { Error } from '@components/Error';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)